docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

```diff
@@ -256,6 +256,14 @@ Allows you to set custom URLs for the Well-Known endpoints.
 
 This is the `redirect_url` which was configured on the Security Token Service (STS).
 
+### `checkRedirectUrlWhenCheckingIfIsCallback`
+
+- Type: `boolean`
+- Required: `false`
+- Default: `false` *NB:* Default will be `true` in v18.
```

Comment:
To keep things consistent with the rest of the docs, the default should be moved to below the description. For example see `silentRenewTimeoutInSeconds`

Reply:
@timdeschryver 
Good point!
Better now? :)

Reply:
Thanks @erichjsonfosse